docs/source/theory/boundaryConditions.rst

```diff
@@ -727,6 +727,28 @@ important for stability for CVFEM tet-based meshes where a natural
 non-orthogonality exists between the boundary and interior integration
 point.
 
+In rare use cases, the usage of the standard open boundary mass flow 
+rate expression, which includes pressure contributions, is not appropriate
+due to complex temperature/buoyancy specifications, e.g., what is commonly
+seen in Bousinesq Atmospheric Boundary Layer (ABL), simulations. In these cases, 
+a global correction algorithm is supported. Specifically, pressure terms are dropped 
+at the open boundary mass flow rate expression
+in favor or a pre-processing algorithm that uniformly distributes the 
+continuity mass flow rate (and possible density accumulation) "error" over
+the entire set of open boundary conditions. This "global corerction" scheme
+may perform well with single open boundary condition specification, e.g., 
+multiple inflows with a single open locaiton, however, it to be avoided if the flow
+leaving the domain is complex in that a simulation includes multiple open boundary
+conditions. A complex situaiton might be an open jet with entrianment from the side 
```

Comment:
sp: situation, entrainment

Reply:
done (sorry, I should have looked at spelling  closer)...